voice-search-widget/InstantSearch.js/public/main.js

```diff
@@ -0,0 +1,85 @@
+import VoiceWidget from "./voice-widget/voice-widget.js";
+
+var socket = io.connect("https://voice-search-demo.herokuapp.com/");
+
+socket.on("connect", function() {});
+
+const search = instantsearch({
+  indexName: "voice_search_demo",
+  searchClient: algoliasearch("XC1DYSAPBX", "3720c40761477e74cb938856acebfa31"),
+  routing: true
+});
+
+search.addWidget(
+  instantsearch.widgets.configure({
+    hitsPerPage: 12
+  })
+);
+
+search.addWidget(
+  instantsearch.widgets.hits({
+    container: "#hits",
+    templates: {
+      empty: "No results",
+      item: `
+            <div class="item">
+                <div class="centered"><img src="{{image}}" alt=""></div>
+                <div class="centered"><div class="add-to-cart"><i class="fas fa-cart-plus"></i> Add <span class="hide-mobile hide-tablet">to Cart</span></div></div>
+                <div class="item-content">
+                    <p class="brand">{{{_highlightResult.brand.value}}}</p>
```

Comment:
This is the way we recommend using highlighting from InstantSearch.js 3:

```suggestion
                    <p class="brand">{{#helpers.highlight}}{ "attribute": "brand" }{{/helpers.highlight}}</p>
```